paper_sections/05-computational_and_statistical_issues.Rmd

```diff
@@ -628,9 +636,21 @@ to balance penalties between nested smooth terms (this is why nested random
+To avoid this issue, we recommend both careful choice of basis and setting model degrees of freedom so that groupwise terms are either slightly less flexible than the global term or have a smaller null space. In the examples in section III, we used smoothers with an unpenalized null space for the global smoother and ones with no null space for the groupwise terms[^gsnull]. When using thin plate splines, this can be done with splines with a lower order of derivative penalized in the groupwise smoothers than the global smoothers, as lower-order "tp" splines have fewer basis functions in the null space. For example, we used `m=2` (penalizing squared second derivatives) for the global smoother, and `m=1` (penalizing squared first derivatives) for groupwise smoothers in models *GS* and *GI*. Another option is to use a lower number of basis functions (`k`) for groupwise relative to global terms.  This will reduce the maximum flexibility possible in the groupwise terms. We do caution that these are just rules of thumb. 
+
+As noted above, interpreting the shape of global terms and group-wise deviations seperately for  *GS* models fit using tensor-product group-level terms is complicated by the fact that **mgcv** will drop some basis-functions for the group-level term to prevent perfect concurvity. 
+For tensor-product smoothers, `gam.side` will generally drop $\le k$ terms from the group-level smoother, where $k$ is the number of basis functions in the global smoother.
```

Comment:
Rather than `gam.side` here (it should be `gam.side()` if used) just use `**mgcv**`. From reading this, the only time that this function is referred to is in the footnote earlier in the section. Just referring to **mgcv** here simplifies what we are trying to say as it is really the behaviour of the package *as encoded in `gam.side`* that is causing this.